src/structures/SimpleMonster.ts

```diff
@@ -77,6 +80,9 @@ export default class SimpleMonster extends Monster {
 				// Not on slayer task
 				loot.add(this.table.roll());
 			}
+			if (this.customKillLogic) {
```

Comment:
This is going to add the customKillLogic loot to the regular kill loot.

It seems that's what you intended, but revenants in game either drop a unique, or their regular table, the uniques aren't a tertiary.